uptime/checker.go

```diff
@@ -0,0 +1,471 @@
+package uptime
+
+import (
+	"context"
+	"sync"
+	"fmt"
+	"time"
+
+	logging "github.com/ipfs/go-log/v2"
+
+	"github.com/filecoin-project/lotus/api/v0api"
+	"github.com/filecoin-project/go-address"
+	chainTypes "github.com/filecoin-project/lotus/chain/types"
+	"github.com/filecoin-project/go-state-types/abi"
+	"github.com/filecoin-project/go-state-types/big"
+
+	"github.com/libp2p/go-libp2p/p2p/protocol/ping"
+	"github.com/libp2p/go-libp2p-core/host"
+	libp2pMultiaddr "github.com/multiformats/go-multiaddr"
+	peerstore "github.com/libp2p/go-libp2p-core/peer"
+)
+
+var log = logging.Logger("uptime")
+
+const NEW_CHECKER_METHOD = 2
+const REPORT_CHECKER_METHOD = 8
+
+const PING_TIMEOUT = 120 // 120 seconds
```

Comment:
why not making these `time.Duration`?
```suggestion
const PING_TIMEOUT = 120 * time.Second // 120 seconds
```